src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<? extends Machine> get() {
+        Machine machine = new Bulldozer();
```

Comment:
There's no need to declare a `Machine` type variable; you can directly create a list of `Bulldozer` objects. Consider creating a local variable of type `List<Bulldozer>` to store the bulldozers and return it.